gui/default/syncthing/transfer/remoteNeededFilesModalView.html

```diff
@@ -0,0 +1,20 @@
+<modal id="remoteNeed" status="info" icon="exchange" heading="{{'Remote Out of Sync Items' | translate}}" large="yes" closeable="yes">
+  <div class="modal-body">
+    <div class="panel panel-default" ng-repeat="folder in remoteNeedFolders">
+      <button class="btn panel-heading" data-toggle="collapse" data-target="#remoteNeed-{{$index}}" aria-expanded="false">
+        <h4 class="panel-title">
+          <span>{{folderLabel(folder)}}</span>
+        </h4>
+      </button>
+      <div id="remoteNeed-{{$index}}" class="panel-collapse collapse">
+        <div class="panel-body">
+          <table class="table table-condensed table-striped">
+            <tr ng-repeat="file in remoteNeed[folder]">
+              <td>{{file.name}}</td>
```

Comment:
I'd add pagination as this list can be in the millions. 
Out of sync dialog has most of that implemented.
We should also do it for failed items.

Reply:
Also more data would be nice.

Reply:
I was considering either grouping and hiding by folders (as done now) or using pagination, but then only sorted by folder. I preferred grouping without pagination, as with pagination it would be difficult to see changes by folder. I'll see whether I find a solution for both. And I will add some more info.

Reply:
You can have pagination within the accordion I guess?